examples/oidc_cli/src/main.rs

```diff
@@ -92,7 +92,7 @@ fn help() {
     println!("Commands:");
     println!("  whoami                 Get information about this session");
     println!("  account                Get the URL to manage this account");
-    println!("  watch [sliding]        Watch new incoming messages until an error occurs");
+    println!("  watch [sliding?]       Watch new incoming messages until an error occurs");
```

Comment:
Hmm, not something for now but can we make sure that we standardize our examples on `clap`. Much nicer to have the commands defined in a declarative manner.